src/main/java/org/isf/patient/manager/PatientBrowserManager.java

```diff
@@ -59,6 +59,14 @@ public class PatientBrowserManager {
 
 	protected LinkedHashMap<String, String> professionHashMap;
 
+	public PatientIoOperations getIoOperations() {
+		return ioOperations;
+	}
+
+	public void setIoOperations(PatientIoOperations ioOperations) {
+		this.ioOperations = ioOperations;
+	}
+
```

Comment:
Why are these two methods here?   The field `ioOPerations` is `@Autowired`; there is no need to set it.

Reply:
yeah, also they seems not used in the whole stack

Reply:
okay i will remove its methods.